en-US/mozorg/home/index-2016.lang

```diff
@@ -90,12 +90,12 @@ Discover unlimited potential, endless possibility and a wide open Web.
 Get Firefox today
 
 
-;Add-ons
-Add-ons
+;Extensions
+Extensions
```

Comment:
This should have raised a huge red flag. Why are you removing an existing string, in an active template? Don't stop at strings when looking at Pull Requests.